sda-download/internal/config/config.go

```diff
@@ -296,6 +308,20 @@ func (c *Map) configArchive() {
 	}
 }
 
+func (c *Map) configReencrypt() {
+	c.Reencrypt.Host = viper.GetString("grpc.host")
+	c.Reencrypt.Port = viper.GetInt("grpc.port")
```

Comment:
This should either have a check that it actually is set or set the default port number.
It should also be aware that if certificates are available the port will be different.

Reply:
Addressed in the commit 8dfc45b. Also, checking whether "grpc.host" is set is done in the parent function `NewConfig()`